package.json

```diff
@@ -4,12 +4,15 @@
   "description": "",
   "main": "server.js",
   "scripts": {
-    "start": "nodemon server.js",
+    "start": "node server.js",
```

Comment:
What was the reason for this? Please change it back - nodemon is a useful package for development purposes.